app/src/utils/chains.ts

```diff
@@ -101,7 +104,7 @@ const POLYGON_TOKENS = [
   { ...DAI_TOKEN, chainId: SupportedChainId.POLYGON, address: '0x8f3Cf7ad23Cd3CaDbD9735AFf958023239c6A063' },
   { ...USDC_TOKEN, chainId: SupportedChainId.POLYGON, address: '0x2791Bca1f2de4661ED88A30C99A7a9449Aa84174' },
   { ...USDT_TOKEN, chainId: SupportedChainId.POLYGON, address: '0xc2132D05D31c914a87C6611C10748AEb04B58e8F' },
-  { ...WBTC_TOKEN, chainId: SupportedChainId.POLYGON, address: '0x1BFD67037B42Cf73acF2047067bd4F2C47D9BfD6' },
+  // { ...WBTC_TOKEN, chainId: SupportedChainId.POLYGON, address: '0x1BFD67037B42Cf73acF2047067bd4F2C47D9BfD6' },
```

Comment:
do we want to remove this ?

Reply:
Yea I commented it out for simplicity / speed for now so we can get this merged. Details in this commit message: https://github.com/dcgtc/dgrants/pull/284/commits/3eb709c55cd273fbf913e47656085e7d0063c3eb